README.md

```diff
@@ -129,14 +129,18 @@ If you have either created your own build of the MailOnline plugin or are replac
 
 #### Using your own rendering code
 
-If you are replacing the MailOnline rendering plugin with your own custom rendering code, you need to change the code where currently the MailOnline plugin is loaded and invoked.
+If you are replacing the MailOnline rendering plugin with your own custom rendering code, you need to change the code where currently an ad renderer is loaded and invoked.
 
+- You have to explicitly set `adRendere` options to `custom` value.
 - Your renderer must be compatible with the Brightcove Player environment.
-- Remove the following function, which is used to load the MailOnline plugin: `./src/BcPrebidVast.js>loadMolPlugin(callback)`
+- Add an function to load your renderer in `./src/BcPrebidVast.js`.
+- If you don't want to use MailOnline plugin, remove the following function, which is used to load the MailOnline plugin: `./src/BcPrebidVast.js>loadMolPlugin(callback)`

```

Comment:
Missing "the" here -- should be: "If you don't want to use **the** MailOnline plugin..."